frontend/integration-tests/views/crud.view.ts

```diff
@@ -121,6 +124,21 @@ export const deleteResource = async(resource: string, kind: string, name: string
   await $('#confirm-action').click();
 };
 
+// Navigates to create new resource page, creates an example resource of the specified kind,
+// then navigates back to the original url.
+export const createNamespacedTestResource = async(kindModel) => {
```

Comment:
Was this taken from `crud.scenario.ts`? Can we refactor to have those tests use this function as well?

Reply:
Yes, parts of this came from `crud.scenario.ts`. There's definitely some overlap between the two. Would it be acceptable to merge this and address the refactor later as a separate issue?